docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -149,6 +159,11 @@ C:\openenclave\build\x64-Debug
 https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
 Normally this is accessible under the `Visual Studio 2017` folder in the Start Menu.
 2. At the x64 Native Tools command prompt, use cmake and ninja to build the debug version:
+To build with DCAP libraries for Windows, add -DUSE_LIBSGX=ON to the cmake command below
+as the default build for Windows is still -DUSE_LIBSGX=OFF i.e. AESM flow.
+However, please ensure that your ACC VM is configured with the DCAP libraries for Windows.
+Current Limitation is that these libraries are installed in c:\openenclave\prereqs\nuget folder.
+Copy or move them to your openenclave directory.
    ```cmd
    cd C:\openenclave
    mkdir build\x64-Debug
```

Comment:
Also fixup other outdate parts of the doc, for example, simulation mode is just explicitly not supported now.

Reply:
Removing the simulation mode temporarily was explicitly addressed by another issue/PR.

Reply:
No, it was not.
- #1784 explicitly disabled simulation mode for windows
- #1647 prior only updated simulation mode instructions for Linux.
- #1770 is still pending specifically in regards to our support statement for simulation mode in general.

This comment was made specifically in regard to:
> Simulation mode is disabled on Windows due to issue #1753.

This should not be in Known Issues as it implies an unfixed bug. We explicitly do not support simulation mode on Windows now. That should be made clear up front in the build instructions.